plugins/modules/kibana_fleet_host.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/python
+# Copyright 2021 Expedient
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -*- coding: utf-8 -*-
+
+#from plugins.modules.ece_cluster import DOCUMENTATION
+
+
+ANSIBLE_METADATA = {
+  'metadata_version': '1.1',
+  'status': ['preview'],
+  'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: kibana_fleet_host
+
+short_description: Add host to kibana fleet
+
+version_added: 2.3.0 # change this
+
+author: Mac Masterson (@mgaruccio)
+
+requirements:
+  - python3
+
+description:
+  - "This module adds a host endpoint to a kibana fleet"
+
+options:
+    host:
+        description:
+            - The Kibana Host you're updating
+        type: str
+    username:
+        description:
+            - Elastic Username
+        type: str
+    password:
+        description:
+            - Elastic Password
+        type: str
+    verify_ssl_cert:
+        description:
+            - Whether or not to verify SSL cert on API requests
+        type: bool
+    url:
+        description:
+            - The url that you want to apply as a fleet server host or an elasticsearch host
+        type: str
+    url_type:
+        description:
+            - The url type that you want to set for the fleet
+            - 'server' sets the fleet server host
+            - 'elasticsearch' sets the fleet elasticsearch host
+        options:
+            - server
+            - elasticsearch
+
+extends_documentation_fragment:
+  - expedient.elastic.elastic_auth_options.documentation
+'''
+
+
+try:
+  from ansible_collections.expedient.elastic.plugins.module_utils.kibana import Kibana
+  import ansible_collections.expedient.elastic.plugins.module_utils.lookups
+except:
+  import sys
+  import os
+  util_path = new_path = f'{os.getcwd()}/plugins/module_utils'
+  sys.path.append(util_path)
+  from kibana import Kibana
+
+from ansible.module_utils.basic import AnsibleModule
+from json import dumps
+
+def main():
+    module_args=dict(
+        host=dict(type='str', required=True),
+        port=dict(type='int', required=True),
+        username=dict(type='str', required=True),
+        password=dict(type='str', required=True, no_log=True),
+        verify_ssl_cert=dict(type='bool', default=True),
+        url_type=dict(type='str', choices=['server', 'elasticsearch'], required=True),
```

Comment:
pedantic, but can we make `server` `fleet_server`?

Reply:
extending that, would it make sense to take a list of `dicts` with `url` and `url_type`?  would probably make it easier to deal with deleting the 443 entries and adding the 9243 ones